install_config/oab_broker_configuration.adoc

```diff
@@ -90,6 +90,15 @@ secrets: <6>
 <5> See xref:oab-config-broker[Broker Configuration] for details.
 <6> See xref:oab-config-secrets[Secrets Configuration] for details.
 
+[[install-config-oab-running-docker]]
+== Running Docker on each node
+
+During installation, ensure that you are running Docker on all nodes of an OpenShift cluster.
+[subs=attributes+]
```

Comment:
I don't think you need the `[subs=attributes+]` here because there aren't any variables, `{variable}`, in use in the code block. This can be changed to `[source,terminal]` though.

Reply:
Looking at the whole file, it doesn't have another instance of `[source,terminal]`, only `[source,yaml]` where necessary. The file must not have been updated when we did a big sweep last year to add `[source,terminal]`. So for consistency sakes, I think you can just remove `[source,terminal]` before the code block and not specify the source language.

Reply:
Updated to remove `[source,terminal]`